### PR TITLE
[VM] Correct wrong syntax of reading a file on az vm help message

### PR DESCRIPTION
**Pull request by haroldrandom**
Fix https://github.com/Azure/azure-cli/issues/9766

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by haroldrandom** [comment]
@qwordy Please take a look

**Comment by qwordy** [comment]
> 
> 
> @qwordy Please take a look

OK

**Comment by haroldrandom** [comment]
help message on SHELL sensitive is planning https://github.com/Azure/azure-cli/issues/8755

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -1884,10 +1884,10 @@
   - name: Update a Windows user account.
     text: az vm user update -u username -p password -n MyVm -g MyResourceGroup
   - name: Update a Linux user account.
-    text: az vm user update -u username --ssh-key-value "$({ ~/.ssh/id_rsa.pub)" -n MyVm -g MyResourceGroup
+    text: az vm user update -u username --ssh-key-value "$(< ~/.ssh/id_rsa.pub)" -n MyVm -g MyResourceGroup
   - name: Update a user on all VMs in a resource group.
     text: >
-        az vm user update -u username --ssh-key-value "$({ ~/.ssh/id_rsa.pub)" --ids $(az vm list -g MyResourceGroup --query "[].id" -o tsv)
+        az vm user update -u username --ssh-key-value "$(< ~/.ssh/id_rsa.pub)" --ids $(az vm list -g MyResourceGroup --query "[].id" -o tsv)
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py]: Does Command Prompt or PowerShell support this syntax? If not, I suggest add a hint on example description. e.g. bash only

**Comment by haroldrandom** [comment]
@MyronFanQiu Please review here

**Pull request merged**